shared/devices/device-page/container.tsx

```diff
@@ -2,15 +2,15 @@ import * as Container from '../../util/container'
 import * as RouteTreeGen from '../../actions/route-tree-gen'
 import DevicePage from '.'
 
-type OwnProps = {}
+type OwnProps = Container.RouteProps<{}, {}>
 
 // TODO(newRouter) after committing to new router:
```

Comment:
[unrelated] We should make a ticket to fix up any of the `todo.*new.*router.*` todos

Reply:
i have a ticket to change all the router2 actions / etc. it'll clean this all up